ratelimit.go

```diff
@@ -171,6 +171,18 @@ func (tb *Bucket) takeAvailable(now time.Time, count int64) int64 {
 	return count
 }
 
+// Available returns the number of available tokens
+func (tb *Bucket) Available() int64 {
```

Comment:
I guess we'd better also doc that `It will be negative when there are consumers waiting for tokens.`